src/Components/Blueprints/ImportBlueprintModal.tsx

```diff
@@ -87,23 +109,36 @@ export const ImportBlueprintModal: React.FunctionComponent<
             browseButtonText="Upload"
             dropzoneProps={{
               accept: { 'text/json': ['.json'] },
-              maxSize: 1024,
+              maxSize: 25000,
               onDropRejected: handleFileRejected,
             }}
-            validated={isRejected ? 'error' : 'default'}
+            validated={isRejected || isInvalidFormat ? 'error' : 'default'}
           />
           <FormHelperText>
             <HelperText>
               <HelperTextItem variant={isRejected ? 'error' : 'default'}>
                 {isRejected
-                  ? 'Must be a JSON file no larger than 1 KB'
+                  ? 'Must be a valid Blueprint JSON file no larger than 25 KB'
+                  : isInvalidFormat
+                  ? 'Not compatible with the blueprints format.'
                   : 'Upload a JSON file'}
               </HelperTextItem>
             </HelperText>
           </FormHelperText>
         </FormGroup>
         <ActionGroup>
-          <Button type="button">Review and finish</Button>
+          <Button
+            type="button"
+            isDisabled={isRejected || isInvalidFormat || !jsonContent}
+            onClick={() =>
+              navigate(resolveRelPath(`imagewizard/import`), {
+                state: { blueprint: importedBlueprint },
+              })
+            }
+            ouiaId="import-blueprint-finish"
+          >
+            Review and finish
```

Comment:
More of a question - do we want to drop the user on the first step or on the review step after importing a blueprint?

Dropping them directly on the Review with all the information prefilled makes more sense to me as it saves the user the work of clicking through the Wizard. Especially since the left side navigation is disabled so there's no quick way to get to the review.

(also I thought the importing doesn't work for a moment, because I didn't notice that there are already some preselected values in the Wizard :sweat_smile:)

What do you think? @avitova @lucasgarfield

Reply:
We discussed this with Kelsea on one of the UI meetings. It makes sense to drop them on the first page. In the future, we will possibly omit some of the data (and also some of the mandatory data) that we allow to export/import, so users would have to go through the wizard anyways:)

Reply:
Got it, makes sense!